src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/RecommendationControllerTest.kt

```diff
@@ -136,6 +137,52 @@ class RecommendationControllerTest() : IntegrationTestBase() {
       .isEqualTo("No recommendation available: No recommendation found for id: 999")
   }
 
+  @Test
+  fun `update a recommendation with alternatives to recall`() {
+    userAccessAllowed(crn)
+    allOffenderDetailsResponse(crn)
+    deleteAndCreateRecommendation()
+    updateRecommendation(updateRecommendationRequest())
+    updateRecommendation(updateRecommendationAlternativesToRecallRequest())
```

Comment:
Not sure you need this? Can we not just do it through the current pattern of putting the update into 'updateRecommendationRequest()' and then triggering 'secondUpdateRecommendationRequest()'? Otherwise we're going to end up with lots of duplicated test data.

Reply:
Neat point, done!